Slim/Utils/Strings.pm

```diff
@@ -623,6 +623,21 @@ sub getLanguage {
 	return $prefs->get('language') || $failsafeLang;
 }
 
+sub isLanguageRegionalVersion {
+	my $lang = shift;
+
+	my $currentLanguage = $prefs->get('language') || $failsafeLang;
+	
+	if ( $currentLanguage =~ /_/ && $lang !~ /_/ ) { # contains a regional variation e.g. EN_GB 
+		my ($mainLang) = split( /_/, $currentLanguage );
+		if ($mainLang eq $lang) {
```

Comment:
You could simplify/perlify this:

```perl
if ( $lang !~ /_/ && $currentLanguage =~ /(.+)_/ ) {
	if ($1 eq $lang) {
		return...
```
No need to apply the regex twice, store the result etc. You can grab the value from the first regex.

I haven't tested this. Please give it a try 😁.

Reply:
Ah yes, thank you.   I've made the change and tested.